src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,192 @@
+                        }
+                        String deadlineStr = inp.substring(9);
+                        String[] inputDeadline = deadlineStr.split("/");
+                        if (inputDeadline.length != 2) {
+                            throw new DukeException("☹ OOPS!!! The description of a deadline must have a date.");
+                        }
+                        String deadLineTaskStr = inputDeadline[0];
+                        String end = inputDeadline[1].substring(3);
+                        Task deadLineTask = new Deadline(deadLineTaskStr, end);
+                        //System.out.println(end);
+                        taskStorage.addTask(deadLineTask);
+                    } catch (DukeException e) {
+                        System.out.println(e.getMessage());
+                    } finally {
+                        break;
+                    }
+
+
+                case "event":
+                    try {
+                        if (inp.length() == 6) {
+                            throw new DukeException("☹ OOPS!!! The description of an event must have a start and end time.");
+                        }
+                        String eventStr = inp.substring(6);
+
+                        String[] eventStrsplit = eventStr.split("/");
+                        if (eventStrsplit.length != 3) {
+                            throw new DukeException("☹ OOPS!!! The description of an event must have a start and end time.");
+                        }
+                        String eventTaskStr = eventStrsplit[0];
+                        String eventBegin = eventStrsplit[1].substring(5);
+                        eventBegin = eventBegin.substring(0, eventBegin.length() - 1);
+                        String eventEnd = eventStrsplit[2].substring(3);
+                        Task eventTask = new Event(eventTaskStr, eventBegin, eventEnd);
+                        taskStorage.addTask(eventTask);
+                    } catch (DukeException e) {
+                        System.out.println(e.getMessage());
+                    } finally {
+                        break;
+                    }
+
+                case "delete":
+                    try {
+                        int taskNo = Integer.parseInt(input[1]);
+                        if (taskNo > taskStorage.noTasks() || taskNo <= 0) {
+                            throw new DukeException("Give a vaild number");
+                        }
+                        Task eventTask = taskStorage.getTask(taskNo - 1);
+                        taskStorage.deleteTask(eventTask);
+                    } catch (NumberFormatException e) {
+                        System.out.println("Number should be typed in");
+                    } catch (DukeException e){
+                        System.out.println(e.getMessage());
+                    } finally {
+                        break;
+                    }
+
+                default:
```

Comment:
good to include default behaviour, consistent with coding standards.

Reply:
Alright thank you!